lib/Doctrine/DBAL/Query/Expression/ExpressionBuilder.php

```diff
@@ -71,6 +71,32 @@ public function andX($x = null)
      *
      * @return CompositeExpression
      */
+    public function or($x = null)
+    {
+        return new CompositeExpression(CompositeExpression::TYPE_OR, func_get_args());
+    }
+
+    /**
+     * @deprecated Use `and()` instead.
+     *
+     * @param mixed $x Optional clause. Defaults = null, but requires
+     *                 at least one defined when converting to string.
+     *
+     * @return CompositeExpression
+     */
+    public function andX($x = null)
+    {
+        return new CompositeExpression(CompositeExpression::TYPE_AND, func_get_args());
```

Comment:
Why duplicate the code instead of calling the new method?

Reply:
It's a one-liner so I hesitated. But you're right. Fixed.

Reply:
The one-liner will show how the new API will need to be called instead of the old one.